overlord/snapstate/snapstate.go

```diff
@@ -485,6 +485,12 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int
 		prev = copyData
 	}
 
+	if !release.OnClassic {
```

Comment:
Should there be multiple conditions here? I don't fully understand all the cases these tasks might get triggered. Right now it's triggered in any case during any install (and refresh/revert?), but all it does is ensuring that the save folder exists on ubuntu-save if the partition is present, so I feel like maybe its ok?